consensus/src/network_interface.rs

```diff
@@ -117,18 +120,49 @@ impl NewNetworkSender for ConsensusNetworkSender {
 }
 
 impl ConsensusNetworkSender {
+    /// Initialize a shared hashmap about connections metadata that is updated by the receiver.
```

Comment:
diff is a little confusing since it is labeled [mempool]

Reply:
True, I'll label it network instead